src/test/v1.2/advanced/fdc3.broadcast.ts

```diff
@@ -824,6 +848,7 @@ export default () =>
           fdc3ApiVersion: "1.2",
           testId: acTestId7,
           notifyAppAOnCompletion: true,
+          appChannelId: testChannel.id,
```

Comment:
This is supposed to be a different channel NOT the same channel - see the test definition in the errorMessage above.